components/treelist/templates/filter.md

```diff
@@ -93,33 +93,24 @@ For example, try filtering with a Min value of 50+ to leave only root-level item
 </TelerikTreeList>
 
 @code {
-    FilterCellTemplateContext theFilterContext { get; set; }
-    public decimal? MinValue { get; set; }
-    public decimal? MaxValue { get; set; }
+    private FilterCellTemplateContext theFilterContext { get; set; }
```

Comment:
Use a capital letter for the variable name. Try using a more descriptive name, instead of "TheFilterDescriptor". For example, "[ColumnName]FilterDescriptor" or something like that